src/constants.js

```diff
@@ -183,3 +183,8 @@ export const MIN_POLLING_INTERVAL = 0.5;
  * Base URL for the tx mining api
  */
 export const TX_MINING_URL = 'https://txmining.mainnet.hathor.network/';
+
+/**
+ * Base URL for the tx mining api in the testnet
+ */
+export const TX_MINING_TESTNET_URL = 'https://txmining.testnet.hathor.network/';
```

Comment:
I feel that we should validate the the tx-mining-service is connected to the same network that we are dealing with.